airflow/www/static/js/dags.js

```diff
@@ -428,17 +442,24 @@ function refreshDagRuns(error, json) {
   });
 }
 
-function handleRefresh() {
+function handleRefresh({ activeDagsOnly = false } = {}) {
+  const dagIds = getDagIds({ activeDagsOnly });
+  const params = new URLSearchParams();
+  dagIds.forEach(((dagId) => {
```

Comment:
```suggestion
  dagIds.forEach((dagId) => {
```
Looks like we have 1 too many parens here

Reply:
Embarrassing, thanks.